test/examples/src/index.md

```diff
@@ -264,7 +264,7 @@ julia> t = T()
 T()
 
 julia> fullname(@__MODULE__)
-(:Main,)
+(:Main, :Main)
```

Comment:
I am unsure how this was *ever* correct... I mean, yeah, ideally it would print `(:Main,)` because that's what is printed in `Main`. But we are in a sandbox module, and we are replacing the module name in output by `Main`, so I think this is kinda the best we can hope for?

C.f.:
```julia-repl
julia> fullname(@__MODULE__)
(:Main,)

julia> module Foo println(fullname(@__MODULE__)) end
(:Main, :Foo)
Main.Foo
```